package lib64icu-devel requires lib64icudata >= 74.2-1, but none of the providers can be installed

**OpenMandriva version**:
Cooker

**Describe the bug**:

```
Run dnf install -y glibc gcc-c++ git gnutar make cmake glib gettext lsb-release rpmdevtools rpm-build rsync glibc-devel boost-devel dbus-devel protobuf-devel protobuf-compiler sqlite-devel libasound-devel pulseaudio-devel lib64GL-devel libgst-plugins-base1.0-devel taglib-devel chromaprint-devel libebur128-devel fftw-devel icu-devel libcdio-devel libgpod-devel libmtp-devel lib64Qt6Core-devel lib64Qt6Concurrent-devel lib64Qt6Network-devel lib64Qt6Sql-devel lib64Qt6DBus-devel lib64Qt6Gui-devel lib64Qt6Widgets-devel lib64Qt6Test-devel qt6-cmake qt6-qtbase-tools qt6-qttools-linguist desktop-file-utils appstream appstream-util hicolor-icon-theme
Last metadata expiration check: 0:00:31 ago on Tue Jan  2 13:[4](https://github.com/strawberrymusicplayer/strawberry/actions/runs/7375599025/job/20093232690#step:5:5)7:14 2024.
Package glibc-6:2.38-[5](https://github.com/strawberrymusicplayer/strawberry/actions/runs/7375599025/job/20093232690#step:5:6).x8[6](https://github.com/strawberrymusicplayer/strawberry/actions/runs/7375599025/job/20093232690#step:5:7)_64 is already installed.
Package lsb-release-3.3-3.noarch is already installed.
Error: 
(try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages)
 Problem: conflicting requests
  - package lib64icu-devel-[7](https://github.com/strawberrymusicplayer/strawberry/actions/runs/7375599025/job/20093232690#step:5:8)4.2-1.x[8](https://github.com/strawberrymusicplayer/strawberry/actions/runs/7375599025/job/20093232690#step:5:9)6_64 from cooker-x86_64 requires lib64icudata >= 74.2-1, but none of the providers can be installed
  - package lib64icu-devel-74.2-1.x86_64 from openmandriva-x86_64 requires lib64icudata >= 74.2-1, but none of the providers can be installed
  - package lib64icudata-74.2-1.x86_64 from cooker-x86_64 requires icu-data = 74.2-1, but none of the providers can be installed
  - package lib64icudata-74.2-1.x86_64 from openmandriva-x86_64 requires icu-data = 74.2-1, but none of the providers can be installed
  - cannot install both icu-data-74.2-1.x86_64 from cooker-x86_64 and icu-data-1:72.1-2.x86_64 from @System
  - cannot install both icu-data-74.2-1.x86_64 from openmandriva-x86_64 and icu-data-1:72.1-2.x86_64 from @System
  - package lib64icudata72-1:72.1-2.x86_64 from @System requires icu-data = 1:72.1-2, but none of the providers can be installed
  - problem with installed package lib64icudata72-1:72.1-2.x86_64
Error: Process completed with exit code 1.
```

**Steps to reproduce**:

`dnf install -y glibc gcc-c++ git gnutar make cmake glib gettext lsb-release rpmdevtools rpm-build rsync glibc-devel boost-devel dbus-devel protobuf-devel protobuf-compiler sqlite-devel libasound-devel pulseaudio-devel lib64GL-devel libgst-plugins-base1.0-devel taglib-devel chromaprint-devel libebur128-devel fftw-devel icu-devel libcdio-devel libgpod-devel libmtp-devel lib64Qt6Core-devel lib64Qt6Concurrent-devel lib64Qt6Network-devel lib64Qt6Sql-devel lib64Qt6DBus-devel lib64Qt6Gui-devel lib64Qt6Widgets-devel lib64Qt6Test-devel qt6-cmake qt6-qtbase-tools qt6-qttools-linguist desktop-file-utils appstream appstream-util hicolor-icon-theme`

**Observed behavior**:

**Expected behavior**:

**Additional comment**:

I use docker image openmandriva/cooker in GH actions and for the past few days I've run into this. Is there are workaround for forcing installing icu-data, or is this a temporary issue on cooker?

**Logs and screenshots if relevant**

--- a/worker-openmandriva-cooker/Dockerfile
+++ b/worker-openmandriva-cooker/Dockerfile
@@ -1,7 +1,6 @@
 FROM openmandriva/cooker
 
-RUN dnf update --assumeyes
-RUN dnf upgrade --assumeyes
+RUN dnf distro-sync --assumeyes
 RUN dnf install --assumeyes \
     glibc gcc-c++ git gnutar make cmake glib gettext lsb-release rpmdevtools rpm-build \
     glibc-devel boost-devel dbus-devel protobuf-devel protobuf-compiler sqlite-devel libasound-devel pulseaudio-devel gnutls-devel lib64GL-devel \

